client/state/sites/reducer.js

```diff
@@ -369,4 +384,5 @@ export default combineReducers( {
 	requesting,
 	hasAllSitesList,
 	jetpackSiteDisconnected,
+	isRequestingJetpackSitesFeatures,
```

Comment:
An opportunity for the future is to try moving those to completely separate Redux root trees. That's the only way we can unblock making the `sites` state tree modular eventually. Currently, it loads for everyone, everywhere, regardless of whether sites are used.